docs/user/build-customization.rst

```diff
@@ -104,6 +105,43 @@ To avoid this, it's possible to unshallow the clone done by Read the Docs:
          - git fetch --unshallow
 
 
+Skip build based on a condition
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+There may be situations where you want to skip a build that was automatically triggered when someone on your team pushed to the repository.
+Skipping builds will allow you to speed up review times and also help us to reduce costs in our servers.
+The following situations are good examples to skip a build:
+
+* the build depends on an external situation that's not met yet
+* there were no changes on the documentation files
+
+In these scenarios, you can skip the build by executing a custom command that checks for that particular condition
+and exits with code ``439`` to skip it, or ``0`` to continue building the documentation normally.
+If any of the commands return this particular exit code,
+Read the Docs will stop the build immediately,
+mark it as "Cancelled",
+and communicate GitHub/GitLab that the build succeeded (green tick) so the pull request is in a mergeable state.
+
+.. code-block:: yaml
+   :caption: .readthedocs.yaml
+
+   version: 2
+   build:
+     os: "ubuntu-22.04"
+     tools:
+       python: "3.11"
+     jobs:
+       post_checkout:
+         # Skip build when there aren't changed in docs directory.
+         # `--quiet` exits with a 1 when there **are** changes,
+         # so we invert the logic with a !
+         #
+         # If there are no changes (exit 0) we force the command to return with 439.
+         # This is a special exit code on Read the Docs that will cancel the build immediately.
+         - ! git diff --quiet origin/main -- docs/ && exit 439
```

Comment:
@humitos as I've noted before in our threads about conditional builds.... doesn't this skip all builds on main itself?

Reply:
Hrm... Good point! I think you are right and this should only be run on pull requests 🤔

Reply:
I updated this code to use an `if` and an environment variable so it only runs on pull requests.

Reply:
I think the new solution looks good :+1: 

It might benefit from one of the multi-line yaml syntax?

```
key: |
   value
   blah
   blah
```

(this syntax preserves the newlines btw)

Reply:
Circle CI uses this syntax a lot :arrow_up:

Reply:
He, I was updating this at the same time you were writing this comment. See [readthedocs/readthedocs.org@`1d4ae24` (#9717)](https://github.com/readthedocs/readthedocs.org/pull/9717/commits/1d4ae249e740e4ee7492f6c54cdb83e23d093f83)

Reply:
We are truly working in the same timezone :smile: